README.rst

```diff
@@ -53,7 +54,10 @@ There are three ways to do this:
   system.
 * Place the file into *C:\\Users\\username\\OpenVPN\\config\\*, or any of its
   immediated subdirectories. The configuration file is only visible for the
-  user in question.
+  user in question. If the user is not a member of "OpenVPN Administrators"
```

Comment:
.... member of the built-in "Administrators" group or "OpenVPN Administrators" group ....